### PR TITLE
Multiple updates

### PR DESCRIPTION
**Pull request by taers232c**
Add member <UserItem> to print cigroups|cigroup-members to select groups to display
Drop Google-Coordinate product ID
Update print|show driveactivity to Drive Activity API v2
Check for more parents than 1 in create|update drivefile
Update documentation

### CODE REVIEW

File: src/gam/__init__.py [hunk 1]
```diff
@@ -3568,6 +3637,14 @@ def getDriveFileAttribute(i, body, parameters, myarg, update=False):
     return i
 
 
+def check_num_parents(user, body):
```
Comment (jay0lee) [inline src/gam/__init__.py:3640]: function name should match what we are returning so we can do something like:

```
if has_multiple_parents(body):
    # warn user
```
and we get an idea of what has_multiple_parents does and returns just by it's name.

Also, let's move the stderr message out of the function and just make the function return true/false. We can output something within the if statement if needed.

File: src/gam/gapi/directory/cros.py [hunk 2]
```diff
@@ -15,11 +16,20 @@
...
 
 
+def _display_cros_command_result(cd, device_id, command_id, times_to_check_status):
+    for _ in range(0, times_to_check_status):
+        time.sleep(2)
+        result = gapi.call(cd.customer().devices().chromeos().commands(), 'get',
+            customerId=GC_Values[GC_CUSTOMER_ID], deviceId=device_id,
+            commandId=command_id)
+        display.print_json(result)
+        if result.get('state') in {'EXPIRED', 'CANCELLED', 'EXECUTED_BY_CLIENT'}:
```
Comment (jay0lee) [inline src/gam/gapi/directory/cros.py:27]:
```
final_states = ['EXPIRED', 'CANCELLED', 'EXECUTED_BY_CLIENT']
if result.get('state') in final_states:
....
```
it's easier to understand what the code is doing here if we declare what we believe are final states ahead of the if statement.

File: src/gam/gapi/directory/cros.py [hunk 3]
```diff
@@ -40,7 +50,7 @@ def issue_command():
              body['commandType'] = command_map[command]
              i += 2
              if command == 'setvolume':
-                 body['payload'] = {'volume': int(sys.argv[i])}
+                 body['payload'] = json.dumps({'volume': sys.argv[i]})
```
Comment (jay0lee) [inline src/gam/gapi/directory/cros.py:54]: you may need to sync to master where this change already exists.

**Comment by taers232c** [comment]
Will re-issue

**Pull request closed without merging**